model/version.go

```diff
@@ -211,6 +211,33 @@ func VersionGetHistory(versionId string, N int) ([]Version, error) {
 	return versions, nil
 }
 
+func (v *Version) UpdateMergeTaskDependencies(p *Project) error {
```

Comment:
Could this race if something else interweaves its dependency updates? Could you use an atomic update, and error (maybe with a very high error level, so we get a slack message) if this happens?
```
	return UpdateOne(
		bson.M{
			IdKey: t.Id,
                         DependsOnKey: <previous state>
		},
		bson.M{
			"$set": bson.M{DependsOnKey: dependsOn},
		},
	)
}
<handle update error>
```

Reply:
done.